readthedocs/search/tests/data/sphinx/out/autodoc.json

```diff
@@ -1,6 +1,6 @@
 {
-  "path": "",
-  "title": "",
+  "path": "autodoc.html",
+  "title": "sphinx.ext.autodoc – Include documentation from docstrings",
```

Comment:
I'm not sure to understand why these are now returned and they weren't before. Can you expand on that?

Reply:
Previously the title and path came from the fjson file, now they come from the file itself.